emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/DynamoDBConstants.java

```diff
@@ -86,4 +89,5 @@
   String DEFAULT_AWS_REGION = Regions.US_EAST_1.getName();
 
   int DEFAULT_AVERAGE_ITEM_SIZE_IN_BYTES = 100;
+  String DEFAULT_CAPCITY_FOR_ON_DEMAND = "40000";
```

Comment:
[nit] typos , should be `DEFAULT_CAPACITY_FOR_ON_DEMAND` and should be using `Long` data type